dist/numjs.js

```diff
@@ -6858,14 +6861,25 @@ NdArray.prototype.sum = function () {
 /**
 * Returns the standard deviation, a measure of the spread of a distribution, of the array elements.
 *
+* @param {boolean} [sample=true]
 * @returns {number}
 */
-NdArray.prototype.std = function () {
+NdArray.prototype.std = function (sample) {
```

Comment:
should rename sample to population here which is actually what's being calculated